tests/test_schema.py

```diff
@@ -45,3 +45,80 @@ def test_relative_path_not_match_schema(data, schema):
     data["rhui_alias"][0]["dest"] = "dest-path"
     with pytest.raises(ValidationError):
         jsonschema.validate(data, schema)
+
+
+def test_empty_list(data, schema):
+    """Verify that YAML returns empty list."""
+    assert data["signing_keys_mappings"]["platform"]["middleware"]["ga_keys"] == []
+
+
+def test_invalid_arches(data, schema):
+    """Verify that data using an invalid arch does NOT match the schema."""
+
+    data["filter_arch_from_releases"]["5"] = "invalid_64"
+    with pytest.raises(ValidationError):
+        jsonschema.validate(data, schema)
+
+
+def test_invalid_signing_key(data, schema):
+    """Verify that data using an invalid signing key does NOT match the schema."""
+
+    data["signing_keys_mappings"]["platform"]["middleware"]["ga_keys"] = "INVALID"
+    with pytest.raises(ValidationError):
+        jsonschema.validate(data, schema)
+
+
+def test_invalid_product_id(data, schema):
+    """Verify that data using an invalid product ID does NOT match the schema."""
+
+    data["ignore_lp_version_product_ids"].append("cats")
+    with pytest.raises(ValidationError):
+        jsonschema.validate(data, schema)
+
+
+def test_ll_tps_stream_string(data, schema):
+    """Verify that data using an invalid TPS stream string does NOT match the schema."""
+    data["rhel_ll_tps_stream_mappings"]["5.1"] = "RHEL-%s-LL-%s"
+    jsonschema.validate(data, schema)
+
+    data["rhel_ll_tps_stream_mappings"]["5.1"] = "RHEL-%s-LongLife-%s"
+    with pytest.raises(ValidationError):
+        jsonschema.validate(data, schema)
+
+
+@pytest.mark.parametrize(
+    "a, b", [("7", "7"), ("5.2", "5.3"), ("7", "5.2"), ("5.2", "7")]
+)
+def test_release_version_strings(data, schema, a, b):
+    """Validate the major_or_minor_version regex."""
+
+    # cfme_version_mappings are a major_or_minor_version to major_or_minor_version mapping.
+    data["cfme_version_mappings"]["platform_full_version"] == a
+    data["cfme_version_mappings"]["platform_version"] == b
+
+
+@pytest.mark.parametrize("major, minor", [("8.0", "8.0"), ("8", "8"), ("8.0", "8")])
+def test_release_version_strings(data, schema, major, minor):
```

Comment:
This test is named identically to the previous one, this probably means only the second is being executed.